plugins/omemo/CMakeLists.txt

```
@@ -67,7 +67,11 @@ GRESOURCES
 add_definitions(${VALA_CFLAGS} -DGETTEXT_PACKAGE=\"${GETTEXT_PACKAGE}\" -DLOCALE_INSTALL_DIR=\"${LOCALE_INSTALL_DIR}\")
 add_library(omemo SHARED ${OMEMO_VALA_C} ${OMEMO_GRESOURCES_TARGET})
 add_dependencies(omemo ${GETTEXT_PACKAGE}-translations)
-target_link_libraries(omemo libdino signal-protocol-vala qrencode ${OMEMO_PACKAGES})
+if(APPLE)
+    target_link_libraries(omemo libdino signal-protocol-vala qrencode ${OMEMO_PACKAGES} ${GCrypt_LIBRARY})
```

Include install directory, so that `qrencode` is found:
```suggestion
    include_directories(${INCLUDE_INSTALL_DIR})
    target_link_libraries(omemo libdino signal-protocol-vala qrencode ${OMEMO_PACKAGES} ${GCrypt_LIBRARY})
```